webapp/channels/src/components/admin_console/feature_discovery/features/auto_translation.scss

```diff
@@ -0,0 +1,23 @@
+@use 'utils/variables';
+
+.AutoTranslationFeatureDiscovery .FeatureDiscovery {
+    all: unset; // removes all inheritable and non-inheritable properties (use with caution)
```

Comment:
Why are we using the same CSS class if we're not inheriting the properties from `.FeatureDiscovery`?

Reply:
basically the inner classes of the FeatureDiscovery component are fine (children) but the parent is causing an additional padding and shadowing that we don't need, so the goal was to use the same component and just override the css so that it fits with I guess this new UI/UX